<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoctorPage</class>
 <widget class="QWidget" name="DoctorPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QTabWidget" name="TabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="AppointmentsTab">
      <attribute name="icon">
       <iconset resource="../../icons.qrc">
        <normaloff>:/Forms/Icons/Forms/Icons/doctor.svg</normaloff>:/Forms/Icons/Forms/Icons/doctor.svg</iconset>
      </attribute>
      <attribute name="title">
       <string>Прием пациентов</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="3" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Действия</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QPushButton" name="AcceptPatient">
            <property name="text">
             <string>Принять</string>
            </property>
            <property name="icon">
             <iconset resource="../../icons.qrc">
              <normaloff>:/Forms/Icons/Forms/Icons/registrator.svg</normaloff>:/Forms/Icons/Forms/Icons/registrator.svg</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QTableWidget" name="AppointmentsTableWidget"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="CardsTab">
      <attribute name="icon">
       <iconset resource="../../icons.qrc">
        <normaloff>:/Forms/Icons/Forms/Icons/card.svg</normaloff>:/Forms/Icons/Forms/Icons/card.svg</iconset>
      </attribute>
      <attribute name="title">
       <string>Карты пациентов</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="3" column="0">
        <widget class="QPushButton" name="AddCardRecordButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Добавить запись</string>
         </property>
         <property name="icon">
          <iconset resource="../../icons.qrc">
           <normaloff>:/Forms/Icons/Forms/Icons/add.svg</normaloff>:/Forms/Icons/Forms/Icons/add.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="RemoveCardRecordButtton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Удалить запись</string>
         </property>
         <property name="icon">
          <iconset resource="../../icons.qrc">
           <normaloff>:/Forms/Icons/Forms/Icons/remove.svg</normaloff>:/Forms/Icons/Forms/Icons/remove.svg</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QTableWidget" name="CardRecordTableWidget"/>
       </item>
       <item row="0" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>ФИО пациента</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="PatientNameField"/>
         </item>
         <item>
          <widget class="QPushButton" name="SearchCardButton">
           <property name="text">
            <string>Открыть карту</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Данные врача</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Специальность:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="SpecialityLabel">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Дата найма:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="EmploymentDateLabel">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections/>
</ui>
